alf/experience_replayers/replay_buffer.py

```diff
@@ -62,6 +63,7 @@ def __init__(self,
                  keep_episodic_info=None,
                  step_type_field="step_type",
                  postprocess_exp_fn=None,
+                 enable_checkpoint=False,
```

Comment:
do we need the ability to turn on or off checkpointing per module?  If this is just for backward compatibility, then the easier way is perhaps to catch any error thrown during loading checkpoint and don't raise it?

Reply:
ReplayBuffer can be huge. Disable it by default to reduce disk usage.